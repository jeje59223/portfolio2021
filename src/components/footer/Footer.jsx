import React from 'react';
import github from '../../images/github.png';
import twitter from '../../images/twitter.png';
import linkedin from '../../images/linkedin.png';
import instagram from '../../images/instagram.png';
import '../../styles/footer.css';

const Footer = () => (
  <footer className="text-center p-4">
    <div className="container">
      <div className="col">
        <p>
          Tous Droits Réservés &copy;
          <span id="year" />
          {' '}
          CNOCKAERT Jérôme
        </p>
        <div className="separate" />
        <p>
          <a href="https://github.com/jeje59223"><img src={github} alt="github" /></a>
          <a href="https://twitter.com/Jeje59223"><img src={twitter} alt="twitter" /></a>
          <a href="https://www.linkedin.com/in/jerome-cnockaert-359a20181/"><img src={linkedin} alt="linkedin" /></a>
          <a href="https://www.instagram.com/cnockaert59223/"><img src={instagram} alt="instagram" /></a>
        </p>
      </div>
    </div>
  </footer>
);

export default Footer;
