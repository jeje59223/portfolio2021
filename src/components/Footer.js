import React from "react";
import github from "../images/github.png";
import twitter from "../images/twitter.png";
import linkedin from "../images/linkedin.png";
import instagram from "../images/instagram.png";
import "../styles/footer.css";

const Footer = () => {
    // let today=new Date();
    // let annee = today.getFullYear();
    // let anneeActuelle = document.getElementById('year');
    // anneeActuelle.innerHTML = annee;
        
    return (
        <footer className="text-center p-4">
        <div className="container">
            <div className="col">
                <p>
                    Tous Droits Réservés &copy;<span id="year"></span> CNOCKAERT Jérôme
                </p>
                <div className="separate"></div>
                <p>
                    <a href="https://github.com/jeje59223"><img src={github} alt="github"/></a>
                    <a href="https://twitter.com/Jeje59223"><img src={twitter} alt="twitter"/></a>
                    <a href="https://www.linkedin.com/in/jerome-cnockaert-359a20181/"><img src={linkedin} alt="linkedin"/></a>
                    <a href="https://www.instagram.com/cnockaert59223/"><img src={instagram} alt="instagram"/></a>
                </p>
            </div>
        </div>
    </footer>
    )
}

export default Footer;