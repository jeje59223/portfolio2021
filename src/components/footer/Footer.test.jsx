import toJson from 'enzyme-to-json';
import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import Footer from './Footer';

configure({ adapter: new Adapter() });

describe('<Footer>', () => {
  it('should render Footer', () => {
    const wrapper = shallow(<Footer />);
    const json = toJson(wrapper);
    expect(json).toMatchSnapshot();
  });
});
