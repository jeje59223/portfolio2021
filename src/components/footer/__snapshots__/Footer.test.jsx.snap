// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> should render Footer 1`] = `
<footer
  className="text-center p-4"
>
  <div
    className="container"
  >
    <div
      className="col"
    >
      <p>
        Tous Droits Réservés ©
        <span
          id="year"
        />
         
        CNOCKAERT Jérôme
      </p>
      <div
        className="separate"
      />
      <p>
        <a
          href="https://github.com/jeje59223"
        >
          <img
            alt="github"
            src="github.png"
          />
        </a>
        <a
          href="https://twitter.com/Jeje59223"
        >
          <img
            alt="twitter"
            src="twitter.png"
          />
        </a>
        <a
          href="https://www.linkedin.com/in/jerome-cnockaert-359a20181/"
        >
          <img
            alt="linkedin"
            src="linkedin.png"
          />
        </a>
        <a
          href="https://www.instagram.com/cnockaert59223/"
        >
          <img
            alt="instagram"
            src="instagram.png"
          />
        </a>
      </p>
    </div>
  </div>
</footer>
`;
